C/common/include/storage_client.h

```diff
@@ -44,6 +46,14 @@ class StorageClient {
 		~StorageClient();
 		ResultSet	*queryTable(const std::string& tablename, const Query& query);
```

Comment:
Why queryTable is not changing? existing queryTable sends a PUT to URL /storage/table , in case , this URL is not going to be supported and replaced with /storage/schema/.../table/... , then this function also needs to be modified

Reply:
This was an oversight, thank you